apps/prairielearn/assets/scripts/lib/questionsTable.js

```diff
@@ -0,0 +1,194 @@
+/* eslint-env browser, jquery */
+
+import _ from 'lodash';
+import { onDocumentReady, decodeData } from '@prairielearn/browser-utils';
+import { html } from '@prairielearn/html';
+
+onDocumentReady(() => {
+  const { course_instance_ids, showAddQuestionButton, urlPrefix, plainUrlPrefix } =
+    decodeData('questions-table-data');
+  window.topicList = function () {
+    var data = $('#questionsTable').bootstrapTable('getData');
+    return _.keyBy(_.map(data, (row) => row.topic.name));
+  };
+
+  window.tagsList = function () {
+    var data = $('#questionsTable').bootstrapTable('getData');
+    return _.keyBy(_.map(_.flatten(_.filter(_.map(data, (row) => row.tags))), (row) => row.name));
+  };
+
+  window.versionList = function () {
+    var data = $('#questionsTable').bootstrapTable('getData');
+    return _.keyBy(_.map(data, (row) => row.display_type));
+  };
+
+  window.qidFormatter = function (qid, question) {
+    var text = '';
+    if (question.sync_errors) {
+      text += html`<button
+        class="btn btn-xs mr-1"
+        data-toggle="popover"
+        data-title="Sync Errors"
+        data-content='<pre style="background-color: black" class="text-white rounded p-3">${_.escape(
+          question.sync_errors_ansified,
+        )}</pre>'
+      >
+        <i class="fa fa-times text-danger" aria-hidden="true"></i>
+      </button>`;
+    } else if (question.sync_warnings) {
+      text += html`<button
+        class="btn btn-xs mr-1"
+        data-toggle="popover"
+        data-title="Sync Warnings"
+        data-content='<pre style="background-color: black" class="text-white rounded p-3">${_.escape(
+          question.sync_warnings_ansified,
+        )}</pre>'
+      >
+        <i class="fa fa-exclamation-triangle text-warning" aria-hidden="true"></i>
+      </button>`;
+    }
+    text += html`<a class="formatter-data" href="${urlPrefix}/question/${question.id}/"
+      >${question.qid}</a
+    >`;
+    if (question.open_issue_count > 0) {
+      text += html`<a
+        class="badge badge-pill badge-danger ml-1"
+        href="${urlPrefix}/course_admin/issues?q=is%3Aopen+qid%3A${_.escape(question.qid)}"
```

Comment:
I'm pretty sure you want [`encodeURIComponent`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent) here, not `_.escape()`.

Reply:
This is just copy pasted from the previous version, but you are probably right, it was probably broken before. I can change it.